tests/console/check_system_info.pm

```diff
@@ -81,6 +81,20 @@ sub check_buildid {
     }
 }
 
+# SLE-21783: Update nodejs-common for SLE15 SP4
+# check the nodejs16 is releaseed to SLE15 SP3 & SLE15 SP4
+# check the nodejs-common points to nodejs16
+# test steps:
+# 1) install nodejs-common on SLES15SP3 and migrated SLES15SP4 when wsm is available in SCC_ADDONS
+# 2) check nodejs16 is installed together with nodejs-common
```

Comment:
I think we need check as 'nodejs-common package needs to be updated to points nodejs-default' while not just zypper se nodejs16.